components/ui/button/utils/helpers.ts

```diff
@@ -186,14 +214,20 @@ export function outlineButtonColors(color?: ButtonColors) {
   }
 }
 
-export function renderButtonIcon(
-  icon?: ButtonProps["icon"],
-  size?: ButtonSizes,
-  additionalClasses?: string,
-) {
+export function renderButtonIcon({
+  icon,
+  size,
+  additionalClasses,
+  iconPosition = "left",
+}: {
+  icon?: ButtonProps["icon"];
+  size?: ButtonSizes;
+  additionalClasses?: string;
+  iconPosition?: ButtonIconPositions;
+}) {
```

Comment:
Changed these parameters to a single object since we're already over 3 parameters now.